books/RayTracingTheNextWeek.html

```diff
@@ -2433,8 +2432,7 @@
 
         color value(double u, double v, const point3& p) const override {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-            auto s = scale * p;
-            return color(1,1,1) * 0.5 * (1 + sin(s.z() + 10*noise.turb(s, 7)));
+            return color(1,1,1)*0.5*(1 + sin(scale * p.z() + 10*noise.turb(p, 7)));
```

Comment:
I personally prefer to keep the spaces between the `*` operands. I would prefer to at least stay consistent across multiplications in a compound statement like this (having `scale * p.z()` and `10*noise.turb(p, 7))` together appears a little jarring to me).

I'll defer to you on the broader style, however.